samples/snippets/src/main/java/com/example/bigtable/deletes/DeleteColumnFamilyExample.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigtable.deletes;
+
+// [START bigtable_delete_column_family]
+import com.google.cloud.bigtable.data.v2.BigtableDataClient;
+import com.google.cloud.bigtable.data.v2.models.RowMutation;
+import java.io.IOException;
+
+public class DeleteColumnFamilyExample {
+  public void deleteColumnFamily(
+      String projectId, String instanceId, String tableId, String rowKey, String familyName) {
+    try (BigtableDataClient dataClient = BigtableDataClient.create(projectId, instanceId)) {
+      dataClient.mutateRow(RowMutation.create(tableId, rowKey).deleteFamily(familyName));
```

Comment:
This is probably clarified by the documentation/reading the code – I think I might've been thrown off by it since I didn't know you can delete all the column family data for a row and defaulting to thinking about deleting CF for the table